app/flame_chart/profile_model.ts

```diff
@@ -27,10 +27,13 @@ export type ThreadEvent = TraceEvent & {
 export type ThreadTimeline = {
   tid: number;
   threadName: string;
+  skyframeNumber?: number;
   events: ThreadEvent[];
   maxDepth: number;
 };
 
+const SKYFRAME_EVALUATOR_PATTERN = /skyframe evaluator (\d+)/;
```

Comment:
this seems a bit brittle, i.e. in the case that bazel changes the thread names used in the profile

you could just simplify the comparator to something like

```parseInt(a.threadName.match( /\d+/g )) - parseInt(b.threadName.match( /\d+/g ))```

Reply:
There's also things like "grpc-command-0" so I think matching just the number is also brittle -- changed the impl to also keep track of the prefix that comes before the number, and for any threads matching the pattern we sort by the number (within the prefix group).